reposilite-site/data/guides/features/dashboard.md

```diff
@@ -3,36 +3,33 @@ id: dashboard
 title: Dashboard
 ---
 
-Reposilite exposes frontend that is mounted to `/#/` as SPA application in Vue 3.
-You can disable frontend in [shared configuration](/guide/settings#shared-configuration)
-if you just want to disable an easy way to query your repository or to provide custom frontend implementation using [static files](/guide/static-files).
+Reposilite exposes a frontend that is mounted to `/#/` as [SPA (_single-page application_)](https://en.wikipedia.org/wiki/Single-page_application) in [Vue 3](https://vuejs.org/). You can disable the frontend in [shared configuration](/guide/settings#shared-configuration) if you want to disable an easy way to query your repository or to provide a custom frontend implementation using [static files](/guide/static-files).
 
 #### Browser
-Browser displays content of currently selected directory, respectively to provided authentication credentials. For not logged user it lists only public files, for authenticated one it also displays files that user has access to.
+Browser displays the content of the currently selected directory in respect to the provided authentication credentials. For users not logged in it lists only public files, for authenticated users it also displays files that the user has access to.
 
 ![Dashboard Browser Preview](/images/guides/dashboard-browser-preview.png)
 
-You can control the way Reposilite displays directory content by opening adjustments view through the icon in right top corner:
+You can control the way how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:
 
 ![Browser Adjustments](/images/guides/dashboard-browser-adjustments.png)
 
 #### Console
 
-CLI *(command-line interface)* displays current Reposilite output and allows to perform available [commands](/guide/standalone#interactive-cli).
-Only access tokens with management permission can access this view.
+CLI (_command-line interface_) displays the current Reposilite output and allows to perform available [commands](/guide/standalone#interactive-cli). Only access tokens with management permission can access this view.
 
 ![CLI Preview](/images/guides/dashboard-console.png)
 
-You can also filter those messages using filters, but keep in mind it searches only though the currently cached log entries, not the whole history.
+You can also filter those messages using filters, but keep in mind that it searches only through the currently cached log entries, not the whole history.
 
 #### Shared configuration
 Shared configuration has been already mentioned in:
 
 * [Guide / Setup :: Shared configuration](/guide/settings#shared-configuration)
 
-Currently, configuration is provided in CDN format:
+Currently, configuration is provided in [CDN (_change detection and Notification_)](https://en.wikipedia.org/wiki/Change_detection_and_notification) format:
```

Comment:
I think `CDN format` is referring to [dzikoysk/cdn](https://github.com/dzikoysk/cdn) (`https://github.com/dzikoysk/cdn`), if so, would it be better to link to that instead?

Reply:
You're right. Wasn't aware of that @dzikoysk developed his own CDN.